addons/web/static/src/views/fields/many2one/many2one_field.js

```diff
@@ -11,13 +11,23 @@ import { Many2XAutocomplete, useOpenMany2XRecord } from "@web/views/fields/relat
 import * as BarcodeScanner from "@web/webclient/barcode/barcode_scanner";
 import { standardFieldProps } from "../standard_field_props";
 
-import { Component, onWillUpdateProps, useState } from "@odoo/owl";
+import { Component, onWillUpdateProps, useState, markup } from "@odoo/owl";
 
 class CreateConfirmationDialog extends Component {
     get title() {
         return sprintf(this.env._t("New: %s"), this.props.name);
     }
 
+    get dialogContent() {
+        return markup(
+            sprintf(
+                this.env._t("Create <strong>%s</strong> as a new %s?"),
+                this.props.value,
+                this.props.name
+            )
+        );
+    }
```

Comment:
```suggestion
        return markup(
                this.env._t("Create <strong>%s</strong> as a new %s?", this.props.value, this.props.name)
        );
    }
```
_t() implements the sprintf logic

Reply:
Not in 16.0